lib/classes/PluginManager.js

```diff
@@ -15,6 +16,33 @@ class PluginManager {
     this.plugins = [];
     this.commands = {};
     this.hooks = {};
+
+    this.autoloadServerlessAlphaPlugin();
+  }
+
+  autoloadServerlessAlphaPlugin() {
+    // get the path to the global node_modules dir
+    // see: https://docs.npmjs.com/files/folders
+    let prefixPath;
+    let globalNodeModulesPath;
+    if (os.platform === 'win32') {
+      const nodeExecPath = path.join(path.sep, 'node.exe');
+      prefixPath = process.execPath.replace(nodeExecPath, '');
+      globalNodeModulesPath = path.join(prefixPath, 'node_modules');
+    } else {
+      const nodeExecPath = path.join(path.sep, 'bin', 'node');
+      prefixPath = process.execPath.replace(nodeExecPath, '');
+      globalNodeModulesPath = path.join(prefixPath, 'lib', 'node_modules');
+    }
+
+    const serverlessAlphaPluginPath = path
+      .join(path.join(globalNodeModulesPath, 'serverless-alpha'));
```

Comment:
Don't know if it's better but we can also utilize something like `Module.globalPaths` to get the global `node_modules` path (see something similar here: https://github.com/serverless/serverless/blob/b57203bc2202b1d69b00ac9fc85ebfb4239f8a52/lib/classes/PluginManager.js#L80)